build/yaml/pipelines/graph.yml

```diff
@@ -7,15 +7,15 @@ trigger:
       - main
   paths:
     include:
-      - packages/MsGraph
```

Comment:
I don't think we should just call these Graph since graph can refer to lots of things. I think this should be MsGraph at a minimum

Reply:
Spoke offline - landed on Graph.